packages/manager/modules/web-universe-components/src/order/order.html

```diff
@@ -0,0 +1,102 @@
+<oui-stepper
+    data-current-index="$ctrl.currentIndex"
+    data-on-finish="$ctrl.validateCheckout()"
+>
+    <div data-ng-transclude></div>
+
+    <oui-step-form
+        data-header="{{:: 'wuc_order_pricing_header' | translate }}"
+        data-editable="$ctrl.pricings.length > 1 && !$ctrl.loading.getCheckoutInformations && !$ctrl.loading.validateCheckout"
+        data-valid="$ctrl.pricing"
+        data-prevent-next="true"
+        data-on-focus="$ctrl.getPricings()"
+    >
+        <div class="mb-3" data-ng-if="$ctrl.hasUniquePricing()">
+            <span
+                data-ng-pluralize
+                data-count="$ctrl.pricings[0].interval"
```

Comment:
It would be better to get the price by `capacities` instead of taking the first one (an issue was brought related to applying this logic: https://github.com/ovh/manager/pull/2076)

Reply:
How ?

Reply:
Within js
```suggestion
find(pricings, { capacities } => capacities.includes(wantedCapacity))             
```
?

Reply:
@marie-j, prices are already taken by capacities. If you take a look at https://github.com/ovh/manager/pull/2072/files#diff-c73c5539b0de80de0d4d05f625869971R92, we pass the `type` property to the component, to filter pricings by capacity. (We choose 'type' word, to describe the type of order).

Let me know if it is ok for you.

Reply:
The main issue is, what if the intervals are different depending on the capacities ?

What is you can order `installation` for 6 months and `renew` for 4, how do you handle this ?